Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -29,12 +29,12 @@
 using NUnit.Framework;
 using Realms;
 using System.ComponentModel;
-using AssemblyToProcess;
 using Mono.Cecil.Cil;
 
 namespace RealmWeaver
 {
     [TestFixture(WeaverOptions.RealmOnlyNoPropertyChanged)]
+    [TestFixture(WeaverOptions.PCLRealmOnlyNoPropertyChanged)]
```

Comment:
I think it'd be nice to have and easy to implement. Besides, we'd _really_ need that when we start weaving .NET Standard assemblies because their surface is even more fragmented than the PCL profile we currently support.